docs/index.rst

```diff
@@ -204,6 +204,24 @@ This command will generate a ``PostFactory`` class that looks like this:
 
     Using ``make:factory --test`` will generate the factory in ``tests/Factory``.
 
+.. tip::
+
+    You can configure globally it which namespace the factories will be generated:
+
+    .. configuration-block::
+
+        .. code-block:: yaml
+
+            # config/packages/zenstruck_foundry.yaml
+            when@dev: # see Bundle Configuration section about sharing this in the test environment
+                zenstruck_foundry:
+                    maker:
+                        factory:
+                            default_namespace: 'App\\MyFactories'
```

Comment:
I'm wondering if this is not too "deep" just for one conf, but it seemed semantically more correct

Reply:
I don't have a strong opinion but what about:

```yaml
zenstruck_foundry:
    make_factory:
        default_namespace: ...
```